services/hackernews/hackernews-user-karma.service.js

```diff
@@ -0,0 +1,64 @@
+import Joi from 'joi'
+import { metric } from '../text-formatters.js'
+import { BaseJsonService, NotFound } from '../index.js'
+import { anyInteger } from '../validators.js'
+
+const schema = Joi.object({
+  karma: anyInteger,
+})
+  .allow(null)
+  .required()
+
+export default class HackerNewsUserKarma extends BaseJsonService {
+  static category = 'social'
+
+  static route = {
+    base: 'hackernews/user-karma',
+    pattern: ':id',
+  }
+
+  static examples = [
+    {
+      title: 'HackerNews User Karma',
+      namedParams: { id: 'drumstick' },
```

Comment:
This example give me 'user not found'. Lets replace it with one that does

```suggestion
      namedParams: { id: 'pg' },
```